ftw/solr/browser/resources/searchpage.scss

```diff
@@ -115,17 +115,26 @@
 }
 
 #selected-facets {
-  @include list();
+  @include list-horizontal();
 
   > li {
     @include list-group-item();
+    border-right: 1px solid $border-color-list-group;
+
+    &:last-child {
+      border-right: 0;
+    }
 
     > .facetRemoveItem {
       @extend .fa-icon;
       @extend .fa-remove;
       padding: 0;
-      display: inline;
+      display: inline-block;
       color: $color-danger;
+      overflow: hidden;
+      width: 1.5em;
+      height: 1.2em;
```

Comment:
If this is for hiding the text next to the `trash` icon, I recommend to use something like this:

``` SCSS
> .facetRemoveItem {
  @extend .fa-icon;
  @extend .fa-remove;
  padding: 0;
  display: inline-block;
  color: $color-danger;
  font-size: 0;

  &:before {
    font-size: em($font-size-base);
  }
}
```